tests/crypto/README.rst

```diff
@@ -0,0 +1,268 @@
+|                    | HMAC SHA512 |                                                                            | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HKDF               | HKDF SHA256 | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HKDF SHA512 |                                                                            | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| EC-JPAKE           | secp256r1   | `J-PAKE: Password-Authenticated Key Exchange by Juggling`_                 | Custom                                                                     |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF9160DK|
+  * |nRF52840DK|
+  * |nRF52DK|
+  * |nRF51DK|
+
+.. note::
+   Nordic devices with smaller flash capacity, such as nRF51, nRF52810, or nRF52811 might not fit the full test suite.
+   A recommended approach in such case is to run the tests one by one.
+
+
+Building and running
+********************
+
+.. |test path| replace:: :file:`tests/crypto/`
+
+.. include:: /includes/build_and_run_test.txt
+
+There are multiple ways to build this test application.
+See :ref:`nrf_security_readme` for additional information about nrf_security configuration.
+You can use the following configuration files to build it in a specific setup:
+
+* :file:`overlay-backend-sw.conf` - software only, except a hardware module to generate entropy for RNG
+* :file:`overlay-backend-hw.conf` - hardware-accelerated using the Arm CryptoCell accelerator (for cryptography and entropy for RNG)
+* :file:`overlay-backend-hw-sw.conf` - a combination of hardware acceleration using Arm CryptoCell and a software implementation that adds key sizes and algorithms not supported in CryptoCell (the setup uses hardware as much as possible)
+
+
+.. _crypto_test_testing:
+
+Testing
+=======
+
+1. Compile and program the application.
```

Comment:
Do we have a link to show how to compile?

Reply:
In the section above, we refer the users to the [Building on the command line](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_programming.html#building-on-the-command-line) section which has this information.
![image](https://user-images.githubusercontent.com/29537761/82900745-46fd4600-9f5d-11ea-9b69-1dfba0e4f423.png)

Unless this is different for tests.